recipes/spidermonkey/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "spidermonkey" %}
+{% set version = "91.8.0" %}
+{% set majorversion = version.split('.')[0] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://releases.mozilla.org/pub/firefox/releases/{{ version }}esr/source/firefox-{{ version }}esr.source.tar.xz
+  sha256: d483a853cbf5c7f93621093432e3dc0b7ed847f2a5318b964828d19f9f087f3a
+  patches:
+    - 0000-fix-config.patch
+    - 0001-va-args.patch
+
+build:
+  skip: true  # [win]
+  number: 0
+  run_exports:
+    - {{ pin_compatible( name|lower ) }}
```

```suggestion
    - {{ pin_subpackage('spidermonkey', max_pin='x') }}
```
The recipe was not pinning the version spec that I expected. Let's try this expression instead.